its/projects/MultipleProjects/WildcardPatternTest/opencover.xml

```diff
@@ -0,0 +1,1520 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
maybe mention in a comment how you modified this file, and which is the source? 

You'll need to put the comment after the `<?xml` tag for it to be valid XML, as far as I remember

see [this example](https://github.com/SonarSource/sonar-dotnet/blob/master/its/projects/CoverageWithDeterministicSourcePaths/opencover.xml#L2)